### ISSUE TITLE
ART - browser search

### ISSUE THREAD

**Issue by lpalbou** [geneontology/noctua-annotation-review#45]
Some users want to use the browser search (Ctrl+F / Cmd+F).

On chrome I didn't have any issue, but it seems on firefox, Karen was not able to search for her PMID which was visible on the page.

My guess is that the focus was not on the correct section of the page, maybe on the left panel ? It's still a very strange behavior. @tmushayahama ideas ?

This search/highlight was only working after clicking on the table part of the page.
<img width="699" alt="Screen Shot 2021-05-06 at 11 33 12 AM" src="https://user-images.githubusercontent.com/24249870/117348255-d8a84f80-ae5e-11eb-82c3-383b020df28a.png">

**Comment by ukemi** [comment]
I just repeated on Firefox making sure my focus was on the main window. It didn't scroll to the term.

**Comment by tmushayahama** [comment]
This is a very special case and not on the todo list for now. I think for now one has to manually scroll to view highlighted text if they wish to use the browser find. This text search is outside the scope of ART or landing page as sometimes.

**Comment by lpalbou** [comment]
Well I am not about over prioritizing but I do believe it’s a fair and probably rather frequent use case so I would prefer Kimberly and David to decide wether it’s a must or nice to have for a v1. 

I believe upon the async loading of the table, and after a search with left panel, you should be able to set the focus there.

I am actually a bit confused as to why this would be needed in the first place and it probably relates to your html layout somehow.

**Comment by ukemi** [comment]
I agree with you both. Although noted, it is certainly not a show stopper and can be in our sights with a lower priority. If we decide to address it I think (you're going to be shocked) that we should do a systematic test of the most common browsers to see which have the behavior and which do not. Then go from there. But I repeat, I think this should be a lower priority item.

**Comment by vanaukenk** [comment]
I don't think this needs to be completed for release of ART v1.
I'd like some more input from @tmushayahama and @lpalbou about the effort that might be needed to address it in a future release, though.
I only test on Chrome, but we've talked in the past about how extensive we should test wrt different browsers.  
In all our Noctua documentation, we should be very clear about what browsers we test and support.
Let's discuss further on today's workbenches call.

**Comment by vanaukenk** [comment]
I added some text to the first page of ART documentation about browser testing and support.  Let's review today.  
https://docs.google.com/document/d/1Yo2O7LWj1wdRGbbOlaxX_AFDl99T-P59JOf5YoebX0Q

**Comment by lpalbou** [comment]
The global testing of ART on multiple browsers should be done indeed and then we can mention in the doc “tested on XX and YY”. Depending of the depth of testing and targeting safari (OS X), Firefox and chrome (OS X + windows) and edge + IE (windows) would probably take around 2 days. 

As for fixing that focus issue, I will let Tremayne speak. IMO not so long as it’s a question of selecting the table container and asking for the browser focus.. but there may be more to it as it’s an unusual bug for me (even out of focus, the search should have worked)

**Comment by vanaukenk** [comment]
Decision made on 2021-05-14 call: we will not address this issue for v1 ART.
@tmushayahama will look into it for a possible future fix.
